web/generate_site/templates/about-base.html

```diff
@@ -0,0 +1,222 @@
+                </div>
+              </div>
+              <p class="mt-3 text-white">
+                {{ org['Biography']|truncate(200, True) }}
+              </p>
+              </div><!-- /.col-lg-4 -->
+            {% endfor %}
+            </div>
+          </div>
+        </div>
+        <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
+          <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+          <span class="sr-only"></span>
+        </a>
+        <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
+          <span class="carousel-control-next-icon" aria-hidden="true"></span>
+          <span class="sr-only"></span>
+        </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="d-md-flex flex-md-equal w-100">
+    <img class="w-100" src="img/about-photo-3.png" />
+  </div>
+
+  <!-- registration details -->
+  <div class="d-md-flex flex-md-equal w-100 my-md-3 ps-md-3">
+    <div class="bg-light me-md-3 py-3 px-3 py-md-5 px-md-5 overflow-hidden">
+      <div class="my-3 py-3 px-5" id="coc">
+        <h5 class="display-6 special-font pink">Code of Conduct</h4>
+        <p class="lead">
+          Welcome to the PyLadies Conference 2023! We are delighted to have you
+          join us for an inspiring and inclusive event aimed at fostering
+          a diverse and vibrant Python community. As advocates for promoting
+          and advancing women's participation in the Python open-source
+          community, we hold the values of respect, inclusivity, and
+          collaboration at the core of our conference. We want to ensure that
+          every participant can experience a positive, safe, and enjoyable
+          environment throughout the event.
+        </p>
+
+        <div class="row w-100">
+          <div id="CoCGroup">
+
+            <button type="button" class="btn btn-primary btn-block mb-2 accordion-btn w-100 coc-button" data-toggle="collapse" data-target="#first">Respect and Inclusivity</button>
+            <div id="first" class="collapse p-3 coc-text" data-parent="#CoCGroup">
+              At the PyLadies Conference, we celebrate diversity and welcome
+              individuals of all backgrounds, regardless of age, race,
+              ethnicity, nationality, gender identity, sexual orientation,
+              religion, or experience level. Everyone deserves to be treated
+              with kindness, empathy, and understanding, and we expect all
+              attendees, speakers, sponsors, and volunteers to uphold this
+              principle.
+            </div>
+
+            <button type="button" class="btn btn-primary btn-block my-3 accordion-btn w-100 coc-button" data-toggle="collapse" data-target="#second">Harassment-Free Environment</button>
+            <div id="second" class="collapse p-3 coc-text" data-parent="#CoCGroup">
+              ...
```

Comment:
@trallard this and the following sections are missing. If you could adapt the PyConUS CoC and update these fields it would be great

Reply:
Would you prefer me to send a PR on top of this one or a follow-up PR to fill in the missing bits?

Reply:
Hi, We need to have the CoC page one way or the other, in order to request PSF sponsorship. What we need is the CoC Description and the Enforcement procedure.
 If creating a PR is easier for you, @trallard please do that.